### ISSUE TITLE
Y axis depth affects weights when parenting bones to mesh

### ISSUE THREAD

**Issue by boringuser** [ndee85/coa_tools#24]
Hi, thank you so much for this tool. It will literally save me hundreds of hours of time!

Apologies if this has already been reported, but I've noticed that the weights that are created when you parent bones to a mesh change depending on the y-axis depth of the mesh. For example, in my .blend file I have a character with an arm in front of the body and another arm in back. I draw the same number of bones over each arm and parent them in the same way, but the weights for the arm in back are different, and the mesh deformation is not correct, whereas the arm in front is. Let me know if you would like me to provide a .blend file showing the end result.

Thanks!

**Comment by boringuser** [comment]
I believe this problem happens because the distance between bones and meshes affects the automatic weights that get assigned during parenting. I was able to avoid this problem by moving my meshes to the same plane on the Y axis before parenting to the armature.

I'm not sure if this is the right way to fix it (I have not worked with the Blender scripting API before) but I changed line 221 in edit_armature.py from this

        bpy.ops.object.parent_set(type='ARMATURE_AUTO')

to this:

        origY = obj.location[1];
        obj.location[1] = 0;
        bpy.ops.object.parent_set(type='ARMATURE_AUTO')
        obj.location[1] = origY;

Basically setting the objects Y axis value to zero before parenting, then putting it back where it was after parenting. Now the weights are consistent, regardless of how the meshes are layered on top of one another.
